lib/datasource/dart/index.ts

```diff
@@ -22,10 +24,7 @@ export async function getPkgReleases({
     body: DartResult;
   } = null;
   try {
-    raw = await got(pkgUrl, {
-      hostType: id,
-      json: true,
-    });
+    raw = await http.getJson(pkgUrl);
```

Comment:
```suggestion
    raw = await http.getJson<DartResult>(pkgUrl);
```

above: `let raw: HttpJsonResponse<DartResult> = null;`